Brixcer_Box/src/main.cpp

```diff
@@ -33,43 +33,40 @@
 
 #define KnobCount 5
 
-#define LEDPIN 13
-
 // Define controls for the box
 KnobControl* knobs = (KnobControl*) malloc(sizeof(KnobControl) * KnobCount);
-//LedControl leds[KnobCount];
+LedControl* leds = (LedControl*) malloc(sizeof(LedControl) * KnobCount);
```

Comment:
I guess there is no tear down / clean up we need to do for this bc it's running on a microcontroller so the TTL is while it has power.

Reply:
Yea, there's not really any time to do cleanup. You unplug the thing, it loses power and can't clean up anyway